mon-pix/app/templates/authenticated/user-account/language.hbs

```diff
@@ -4,4 +4,10 @@
     @selectedLanguage={{@model.lang}}
     @onLanguageChange={{this.onLanguageChange}}
   />
+
+  {{#if this.showLanguageUpdatedMessage}}
```

Comment:
**Question** : `showLanguageUpdatedMessage` ou `should(Show|Display)LanguageUpdatedMessage` ?

Reply:
Vu le dernier atelier standards d'équipe je pencherai pour `shouldDisplayLanguageUpdatedMessage`